### PR TITLE
UPDATE: README.md & docs

### PR DESCRIPTION
**Pull request by nicolas2bert**
S3 Server becomes Zenko CloudServer!

### CODE REVIEW

File: docs/ARCHITECTURE.rst [hunk 1]
```diff
@@ -217,11 +217,11 @@ A deletion targeting the latest version of an object has to:
     -  if no more versions exist, metadata deletes the master version,
        removing the key from metadata
 
-Note: all of this happens before responding to S3, and only when the
-metadata engine is instructed by S3 to delete a specific version or the
-master version. See section `"Delete Markers" <#delete-markers>`__ for a
-description of what happens when a Delete Object request is sent to the
-S3 API.
+Note: all of this happens before responding to Zenko CloudServer, and only when
```
Comment (electrachong) [inline docs/ARCHITECTURE.rst]: * all of this happens in metadata before responding to the front-end api,

Comment (nicolas2bert) [inline docs/ARCHITECTURE.rst]: ok - thanks

File: README.md [hunk 2]
```diff
@@ -1,19 +1,33 @@
-# Scality S3 Server
+# Zenko CloudServer
```
Comment (LaurenSpiegel) [inline README.md]: why not Zenko Kumo (means cloud in Japanese)?

**Comment by GiorgioRegni** [comment]
Enterprise passed, public ci fails due to socket hangup

**Pull request merged**